frontend/src/pages/Player/Toolbar/TimelineIndicators/TimelineIndicatorsBarGraph/TimelineIndicatorsBarGraph.tsx

```diff
@@ -276,6 +277,17 @@ const TimelineIndicatorsBarGraph = ({
 						eventType: 'Errors',
 					} as SessionEvent),
 			),
+			...(rageClicks.map((rageClick) => ({
+				relativeIntervalPercentage: rageClick.startPercentage,
+				timestamp: toTS(rageClick.startPercentage),
```

Comment:
makes sense to place the rage clicks at their start, though they would span through the end of the range. cc @julian-highlight who is thinking about improving the UI here as a follow up, to show the span of the range.

Reply:
Yep this will come soon! For the short term the starting point works perfectly, as it also doesn't need us to change anything on the layout/add something to the layout. 

For Q2 we plan to add a view that is able to display time spans! Currently all designs are rather experimental! 

@Vadman97 @betty-alagwu

Reply:
Thank you @julian-highlight for the update.